### PR TITLE
[docs] Cloud Select database version

### PR DESCRIPTION
**Pull request by ddhodge**
Add select database version feature on cluster creation, and cluster creation/planning overview.

@netlify /latest/yugabyte-cloud/cloud-basics/

### CODE REVIEW

File: docs/content/latest/yugabyte-cloud/cloud-basics/create-clusters.md [hunk 1]
```diff
@@ -100,19 +100,19 @@ Download the credentials, and click **Create Cluster**.
 
 {{< warning title="Important" >}}
 
-Save your credentials in a safe place. If you lose these credentials you will not be able to use the database.
+Save your database credentials in a safe place. If you lose these credentials you will not be able to use the database.
```
Comment (polarweasel) [inline docs/content/latest/yugabyte-cloud/cloud-basics/create-clusters.md:103]:
```suggestion
Save your database credentials in a safe place. If you lose these credentials, you won't be able to use the database.
```

File: docs/content/latest/yugabyte-cloud/cloud-basics/create-clusters.md [hunk 2]
```diff
@@ -100,19 +100,19 @@ Download the credentials, and click **Create Cluster**.
...
 
 {{< /warning >}}
 
 ## Viewing the cluster
 
-After you complete the wizard, the [**Clusters**](../../cloud-clusters/) page appears with the provisioning of your new cluster in progress.
+After you complete the wizard, the **Clusters** page appears with the provisioning of your new cluster in progress.
```
Comment (polarweasel) [inline docs/content/latest/yugabyte-cloud/cloud-basics/create-clusters.md:109]:
```suggestion
After you complete the wizard, the **Clusters** page appears, showing the provisioning of your new cluster in progress.
```

**Review by polarweasel — CHANGES_REQUESTED** [review]
Looking good, just some little stuff...

File: docs/content/latest/yugabyte-cloud/cloud-basics/create-clusters-overview.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+For lowest data transfer costs, use the same cloud provider and locate your cluster in the same region as the applications that will be connecting to the cluster. For a list of supported regions, refer to [Cloud provider regions](../../release-notes/#cloud-provider-regions).
+
+## Sizing
+
+The basic configuration for Yugabyte Cloud clusters includes 2 vCPUs per node. Each vCPU comes with 50GB of storage. A node has a minimum of 2 vCPUs.
+
+Depending on your performance requirements, you can increase the number of vCPUs per node, as well as the total number of nodes. For production clusters, a minimum of 3 nodes with 2 vCPUs per node is recommended.
+
+Yugabyte Cloud supports both vertical and horizontal scaling, so you can change these values after the cluster is created if your configuration doesn't match your performance requirements. This includes increasing or decreasing vCPUs and increasing storage, and adding or removing nodes. Refer to [Scaling clusters](../../cloud-clusters/configure-clusters/).
```
[the remark this answers is not in the document]
Comment (ddhodge) [reply docs/content/latest/yugabyte-cloud/cloud-basics/create-clusters-overview.md:30]: Ratchet

**Pull request merged**